src/plugins/aznamer.ts

```diff
@@ -1,37 +1,56 @@
 import { CodeModel, codeModelSchema } from '@azure-tools/codemodel';
-import { Session, startSession, Host } from '@azure-tools/autorest-extension-base';
+import { Session, startSession, Host, Channel } from '@azure-tools/autorest-extension-base';
 import { serialize, deserialize } from '@azure-tools/codegen';
 import { values, items, length, Dictionary } from '@azure-tools/linq';
 
 class AzNamer {
-    codeModel: CodeModel
+    codeModel: CodeModel;
 
     constructor(protected session: Session<CodeModel>) {
         this.codeModel = session.model;
     }
 
-    async init() {
-        // any configuration if necessary
-        return this;
+    changeCamelToDash(str: string) {
```

Comment:
we should add this in some kind of common helpers file for entire extension.
doesn't such function exist in any common autorest libraries?

Reply:
I agree.